src/util-file.h

```diff
@@ -25,14 +25,23 @@
 #ifndef __UTIL_FILE_H__
 #define __UTIL_FILE_H__
 
-#ifdef HAVE_NSS
-#include <sechash.h>
-#endif
-
 #include "conf.h"
 
 #include "util-streaming-buffer.h"
 
+/* Hack: Pulling rust.h to get the SCSha256 causes all sorts of problems with
```

Comment:
Any ideas for resolving this?

Reply:
No, spent a lot of time trying to get `rust.h` included here but it just leads to too many conflicts.. I think because this is included in detect.h, but the Rust headers depends on some app-layer stuff. Maybe its something some forward declarations in just the right spot could solve? Can revisit when the rest this is OK, as like a said, a lot of time spent here so far with no gain.